draft-ietf-ccamp-yang-otn-slicing.md

```diff
@@ -160,8 +160,8 @@ normative:
 {: #tab-prefixes title="Prefixes and Corresponding YANG Modules"}
 
 RFC Editor Note:
-Please replace VVVV with the RFC number assigned to {{?I-D.ietf-teas-ietf-network-slice-nbi-yang}}.
-Please replace WWWW with the RFC number assigned to {{?I-D.ietf-teas-rfc8776-update}}.
+Please replace VVVV with the RFC number assigned to {{!I-D.ietf-teas-ietf-network-slice-nbi-yang}}.
+Please replace WWWW with the RFC number assigned to {{!I-D.ietf-teas-rfc8776-update}}.
```

Comment:
Actually RFC8776-bis is not used in this draft while the NS Topology model is used but not referenced int the table above

```suggestion
Please replace WWWW with the RFC number assigned to {{!I-D.ietf-teas-network-slice-topology-yang}}.
```

Also update the table above:

OLD

`
   | te-types | ietf-te-types                | \[RFCWWWW]        |
`

NEW

`
   | ns-topo | ietf-ns-topo                | \[RFCWWWW]        |
`